tests/client/amo/reducers/testLanding.js

```diff
@@ -52,11 +52,11 @@ describe('landing reducer', () => {
         addonType, featured, highlyRated, loading, popular,
       } = landing(initialState, getLanding({ addonType: ADDON_TYPE_THEME }));
 
-      assert.equal(addonType, ADDON_TYPE_THEME);
-      assert.equal(loading, true);
-      assert.deepEqual(featured, { foo: 'bar' });
-      assert.deepEqual(highlyRated, { count: 0 });
-      assert.deepEqual(popular, { results: [] });
+      expect(addonType).toEqual(ADDON_TYPE_THEME);
+      expect(loading).toEqual(true);
```

Comment:
This won't be a problem but strictly speaking single value could use toBe it might be faster for starters.

In any case, for something like this if loading was 'blah' you'd get:

```
Comparing two different types of values. Expected boolean but received string.
```